projects/torvalds/linux/Makefile

```diff
@@ -3,7 +3,7 @@ GIT_TAG=$(shell cat GIT_TAG)
 
 REPO=linux
 REPO_OWNER=torvalds
-REPO_SPARSE_CHECKOUT=tools lib/bootconfig.c include/linux/bootconfig.h
+REPO_SPARSE_CHECKOUT=COPYING tools lib/bootconfig.c include/linux/bootconfig.h
```

Comment:
this is the only other project using this license method. while looking at this i noticed since we sparse checkout this repo we werent getting the right license file. this fixes that.